recipes/pillow/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: pillow
+  version: 3.2.0
+
+source:
+  fn: Pillow-3.2.0.tar.gz
+  url: https://github.com/python-pillow/Pillow/archive/3.2.0.tar.gz
+  sha256: eb4b9c2956e38f89d588a0d0af0e27a03f1e1d2be705a828bbd12cfba0b020fa
+
+build:
+  number: 0 
+  script: pip install --no-deps .
+
+requirements:
+  build:
+    - python
+    - pip
+    - zlib 1.2*
+    - libtiff 4*
+    - jpeg 8*
```

Comment:
@pelson what was the reason to choose the older `jpeg 8` instead of the new `jpeg 9b`? (See https://github.com/conda-forge/rasterio-feedstock/issues/5)